gnucash/gtkbuilder/dialog-preferences.glade

```diff
@@ -2021,6 +2021,25 @@ many months before the current month:</property>
                     <property name="top_attach">1</property>
                   </packing>
                 </child>
+                <child>
+                  <object class="GtkCheckButton" id="pref/general.register/type-ahead-search">
+                    <property name="label" translatable="yes">_Use type ahead</property>
+                    <property name="visible">True</property>
+                    <property name="can_focus">True</property>
+                    <property name="receives_default">False</property>
+                    <property name="has_tooltip">True</property>
+                    <property name="tooltip_markup">Use the type-ahead search for accounts</property>
+                    <property name="tooltip_text" translatable="yes">Use the type-ahead search for accounts</property>
```

Comment:
I know, most tooltips are currently of this kind, but better would be a short explanation, which a new user would understand:
"The normal behaviour is ... \n
Type ahead search allows to ..."

Reply:
100% agreed. I'll improve it, thanks for the suggestion.